tests/Oryx.BuildImage.Tests/Node/NodeDynamicInstallationTest.cs

```diff
@@ -0,0 +1,65 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Oryx.Common;
+using Microsoft.Oryx.Tests.Common;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Oryx.BuildImage.Tests
+{
+    public class NodeDynamicInstallationTest : NodeJSSampleAppsTestBase
+    {
+        private readonly string DefaultInstallationRootDir = "/opt/nodejs";
+
+        public NodeDynamicInstallationTest(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        public void DynamicallyInstallsNodeRuntimeAndBuilds()
+        {
+            // Arrange
+            var devPackageName = "nodemon";
```

Comment:
add a comment may be? that nodemon and express is present in sampleapp's package.json